src/components/BottomBar/index.js

```diff
@@ -16,10 +15,6 @@ import { getShortLocalDateTimeString } from '../../helpers/utility';
 @inject('store')
 @observer
 export default class BottomBar extends Component {
-  static propTypes = {
-    classes: PropTypes.object.isRequired,
```

Comment:
this is required. the `Paper` in the render function uses `classes`.

Reply:
oh ok, I thought we are going to remove all propTypes, aren't we?

Reply:
@amazingandyyy classes is a proptype from the material UI lib. its not something that would go into the store. same goes for things like localization proptypes, etc.